### PR TITLE
BugFix: Fix issues with dispatches that occur within 3 minutes of each other

### PR DESCRIPTION
**Pull request by CKegel**

### CODE REVIEW

File: public/js/whiteboard.js [hunk 1]
```diff
@@ -151,6 +153,9 @@ function clearDispatch() {
 }
 
 function handleDispatch(determinant, complaint, location) {
+  clearDispatch();
+  clearTimeout(dispatchTimeoutID);
```
Comment (ddbruce) [inline public/js/whiteboard.js:157]: Shouldn't this be
```
if (dispatchTimeoutID != "")  {
    clearTimeout(dispatchTimeoutID);
    dispatchTimeoutId = "";
}
```
?

Comment (CKegel) [reply public/js/whiteboard.js:157]: That is most definitely more correct if not correct.

Comment (CKegel) [reply public/js/whiteboard.js]: > Shouldn't this be
> 
> ```
> if (dispatchTimeoutID != "")  {
>     clearTimeout(dispatchTimeoutID);
>     dispatchTimeoutId = "";
> }
> ```
> 
> ?

I've made the appropriate changes to reflect this corrected logic

File: public/js/whiteboard.js [hunk 2]
```diff
@@ -151,6 +154,11 @@ function clearDispatch() {
 }
 
 function handleDispatch(determinant, complaint, location) {
+  if(dispatchTimeoutID != ""){
```
Comment (ddbruce) [inline public/js/whiteboard.js:157]: Please add a space after `if` and before `(`

Comment (ddbruce) [reply public/js/whiteboard.js:157]: And after `)` and before `{`

Comment (ddbruce) [reply public/js/whiteboard.js:157]: ESLint should take care of these things. Are you using that?

**Review by ddbruce — CHANGES_REQUESTED** [review]

**Comment by CKegel** [comment]
An Authentication token has been added for the dispatch endpoint

**Review by ddbruce — APPROVED** [review]
lgtm

**Pull request merged**